docs/cli.md

```diff
@@ -0,0 +1,744 @@
+    <img src="images/cli004.png" class="invert-colors-in-dark-mode" alt="ls cli example"/>
+  </td>
+  </tr>
+</table>
+
+
+### mkdir
+`mkdir` is short for "make directory". It creates a directory with the specified name.
+[More info](https://man7.org/linux/man-pages/man1/mkdir.1.html).
+
+CLI Example:
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ ls
+  example.txt main.cpp stuff
+  $ mkdir stuff2
+  $ ls
+  example.txt main.cpp stuff stuff2
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli005.png" class="invert-colors-in-dark-mode" alt="ls cli example"/>
+  </td>
+  </tr>
+</table>
+
+GUI Equivalent:
+
+<img src="images/cli012.png" class="invert-colors-in-dark-mode" width="768px" alt="mkdir gui example"/>
+
+### touch
+`touch` creates a file with the specified name.
+[More info](https://man7.org/linux/man-pages/man1/touch.1.html).
+
+CLI Example:
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ ls
+  example.txt main.cpp stuff stuff2
+  $ touch euchre.cpp
+  $ ls
+  example.txt main.cpp stuff stuff2 euchre.cpp 
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli006.png" class="invert-colors-in-dark-mode" alt="ls cli example"/>
+  </td>
+  </tr>
+</table>
+
+
+### rm
+`rm` is short for "remove". It deletes the file or directory at the specified path.
+
+You must include the `-r` (recursive) <b>option</b> to delete a directory.
+
+CLI Example:
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ ls
+  euchre.cpp example.txt main.cpp stuff stuff2
+  $ rm euchre.cpp
+  $ ls
+  example.txt main.cpp stuff stuff2
+  $ rm stuff2
+  rm: stuff2: is a directory
+  $ rm -r stuff2
+  $ ls
+  example.txt main.cpp stuff
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli007.png" class="invert-colors-in-dark-mode" alt="ls cli example"/>
+  </td>
+  </tr>
+</table>
+
+GUI Equivalent:
+
+<img src="images/cli013.png" class="invert-colors-in-dark-mode" width="768px" alt="rm gui example"/>
+
+
+### cd
+`cd` stands for "change directory". It switches you to the specified directory. [More info](https://man7.org/linux/man-pages/man1/cd.1p.html).
+
+CLI Example:
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ pwd
+  /Users/ohjun/Desktop/project
+  $ ls
+  example.txt main.cpp stuff
+  $ cd stuff/
+  $ ls
+  $ 
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli008.png" class="invert-colors-in-dark-mode" alt="ls cli example"/>
+  </td>
+  </tr>
+</table>
+
+GUI Equivalent:
+
+<img src="images/cli014.png" class="invert-colors-in-dark-mode" width="768px" alt="cd gui example"/>
+
+
+### mv
+`mv` is short for "move". It moves the specified file or directory to the specified path.
+
+`mv` is also commonly used to rename a file or directory. [More info](https://man7.org/linux/man-pages/man1/mv.1.html).
+
+CLI Example (moving):
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ ls
+  example.txt main.cpp stuff
+  $ mv main.cpp stuff/
+  $ ls
+  example.txt stuff
+  $ ls stuff/
+  main.cpp
+  ```
+
+  </td>
+  <td>
```

Comment:
How about a single image that captures the dragging part of drag-and-drop?  Then, write on the image "drag and drop".